tests/end2end/features/api/print.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Print to STDOUT
+
+    Scenario: Print trivial list
+        Given I start vimiv
+        And I capture output
+        When I run print-stdout "test"
+        Then stdout should contain 'test'
+
+    Scenario: Print longer list
+        Given I start vimiv
+        And I capture output
+        When I run print-stdout "test1", "test2"
+        Then stdout should contain 'test1'
+        And stdout should contain 'test2'
+
+    Scenario: Print list using seq
```

Comment:
seq -> sep :blush:

Reply:
fixed in b039b82bcb453c33673e671b3d9d2ef7c306f7fa